addons/website/views/snippets/snippets.xml

```diff
@@ -4,7 +4,7 @@
 <!-- Snippets menu -->
 <template id="snippets" inherit_id="web_editor.snippets" primary="True" groups="base.group_user">
     <xpath expr="//button[@data-action='cancel']" position="before">
-        <button type="button" class="btn btn-secondary" data-action="mobile" title="Mobile Preview" accesskey="v"><span class="fa fa-mobile" style="font-size: 18px;"/></button>
+        <button type="button" class="btn btn-secondary" data-action="mobile" title="Mobile Preview" accesskey="v"><span class="fa fa-mobile" style="font-size: 20px;"/></button>
```

Comment:
The comment I made for the same button outside of edit is true for this one too. It should be a shared CSS rule or at worst a shared CSS variable value to ensure they have the same font-size always.